contrib/android-tools/template.py

```diff
@@ -0,0 +1,32 @@
+pkgname = "android-tools"
+pkgver = "34.0.4"
+pkgrel = 0
+build_style = "cmake"
+hostmakedepends = ["cmake", "ninja", "perl", "go", "pkgconf", "protobuf"]
+makedepends = [
+    "brotli-devel",
+    "lz4-devel",
+    "pcre2-devel",
+    "libusb-devel",
+    "zstd-devel",
+    "protobuf-devel",
+    "linux-headers",
+    "gtest-devel",
+]
+depends = ["python", "android-udev-rules"]
+pkgdesc = "Android platform tools (e.g. adb and fastboot)"
+maintainer = "Jami Kettunen <jami.kettunen@protonmail.com>"
+license = "Apache-2.0 AND ISC AND GPL-2.0-only AND MIT"
+url = "https://github.com/nmeum/android-tools"
+source = f"{url}/releases/download/{pkgver}/android-tools-{pkgver}.tar.xz"
+sha256 = "7a22ff9cea81ff4f38f560687858e8f8fb733624412597e3cc1ab0262f8da3a1"
+hardening = ["vis", "cfi"]
```

Comment:
i'd be a little surprised if this actually worked with cfi (not doubting you; just a cautious double check of all the tooling)

Reply:
`adb` and `fastboot` for my typical usage at least didn't crash and burn, but they have so much functionality that I'm not sure it's even possible to properly test it all with a single device (as some have certain fastboot commands not implemented at all on bootloader level etc)

Maybe it's just better to forget about CFI to avoid potentially bricking devices during critical operations with a CFI-related crash even though so far it "appears to be fine"

Reply:
that sounds like a realistic enough test if you want to keep it on :)  

if you think it has the potential to brick devices and whatnot then it would be safer to disable it, yes. the hardening itself is probably of 'lesser' value because like most hardening approaches, it has most value for longrun services, things with privileges, ... . these commands are basically user tools, so it's always just a small nice to have i guess, when it's safe

Reply:
it's c++, it's most likely safe